src/chk/chk_internal.h

```diff
@@ -40,8 +46,119 @@ enum chk_rpc_opc {
 
 #undef X
 
-int chk_rejoin(void);
+/* dkey for check DB under sys_db */
+#define CHK_DB_TABLE		"chk"
 
-int chk_pause(void);
+/* akey for leader bookmark under CHK_DB_TABLE */
+#define CHK_BK_LEADER		"leader"
+
+/* akey for engine bookmark under CHK_DB_TABLE */
+#define CHK_BK_ENGINE		"engine"
+
+/* akey for check property under CHK_DB_TABLE */
+#define CHK_PROPERTY		"property"
+
+#define CHK_BK_MAGIC_LEADER	0xe6f703da
+#define CHK_BK_MAGIC_ENGINE	0xe6f703db
+#define CHK_BK_MAGIC_POOL	0xe6f703dc
+
+/*
+ * XXX: Please be careful when change CHK__CHECK_INCONSIST_CLASS__CIC_UNKNOWN
+ *	to avoid hole is the struct chk_property.
```

Comment:
It's bit difficult to follow this comment, not sure how this CHK__CHECK_INCONSIST_CLASS__CIC_UNKNOWN is relating to chk_property.

Reply:
The CHK__CHECK_INCONSIST_CLASS__CIC_UNKNOWN is related with the chk_property sizey.. If it is changed improperly, there will be hold in chk_property.